app/screens/thread/thread_post_list/index.ts

```diff
@@ -44,6 +45,9 @@ const enhanced = withObservables(['channelId', 'forceQueryAfterAppState', 'rootP
                 return queryPostsChunk(database, rootPost.id, earliest, latest, true).observe();
             }),
         ),
+        teamId: rootPost.channel.observe().pipe(
+            switchMap((channel) => of$(channel?.teamId)),
```

Comment:
why is channel conditional?

Reply:
as `channel` could be `ChannelModel` or `null`. TS was throwing the error as well.